core/conversion/converters/impl/softmax.cpp

```diff
@@ -26,12 +26,17 @@ static auto softmax_registrations TRTORCH_UNUSED = RegisterNodeConversionPattern
        }
 
        int64_t dim = args[1].IValue()->toInt();
+       LOG_DEBUG("Softmax original dim " << dim);
+       if (dim == -1) {
+         dim = shape.size() - 1;
```

Comment:
can we extend it to handle all negative indices ? `dim = shape.size()  + dim` for dim < 0 or will that case never happens ?

Reply:
Yeah good change